script.js

```diff
@@ -9,59 +11,98 @@ const buttonAdd = document.querySelector('.button-add');
 const displayBooks = document.getElementById('books-list');
 const bookShelf = [];
 
-// Storage functions
+// Storage classes
+class Store {
+  static storeGet() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
 
-function storeGet() {
-  let books;
-  if (localStorage.getItem('books') === null) {
-    books = [];
-  } else {
-    books = JSON.parse(localStorage.getItem('books'));
+    return books;
   }
 
-  return books;
-}
-function storeAdd(newBook) {
-  const books = storeGet();
-  books.push(newBook);
-  localStorage.setItem('books', JSON.stringify(books));
-}
-function storeDelete(title) {
-  const books = storeGet();
-  const index = books.findIndex((book) => book.title === title);
-  if (index !== -1) {
-    books.splice(index, 1);
+  static storeAdd(newBook) {
+    const books = Store.storeGet();
+    books.push(newBook);
     localStorage.setItem('books', JSON.stringify(books));
   }
+
```

Comment:
- [OPTIONAL] 🤔 It will be better to empty the book title and book author after adding a book to the list.